pkg/ccl/backupccl/backup_destination.go

```diff
@@ -344,13 +344,12 @@ func readLatestFile(
 func findLatestFile(
 	ctx context.Context, exportStore cloud.ExternalStorage,
 ) (ioctx.ReadCloserCtx, error) {
-	latestFile, err := exportStore.ReadFile(ctx, latestHistoryDirectory+"/"+latestFileName)
+	latestFile, err := exportStore.ReadFile(ctx, backupMetadataDirectory+"/"+latestHistoryDirectory+"/"+latestFileName)
```

Comment:
can we just define `latestHistoryDirectory` var as `backupMetadataDirectory/latest`. I think it is easier to read that way.